Zend/zend_builtin_functions.c

```diff
@@ -140,9 +140,13 @@ ZEND_FUNCTION(gc_status)
 
 	array_init_size(return_value, 3);
 
+	add_assoc_bool_ex(return_value, "active", sizeof("active")-1, status.active);
+	add_assoc_bool_ex(return_value, "protected", sizeof("protected")-1, status.gc_protected);
+	add_assoc_bool_ex(return_value, "full", sizeof("full")-1, status.full);
 	add_assoc_long_ex(return_value, "runs", sizeof("runs")-1, (long)status.runs);
 	add_assoc_long_ex(return_value, "collected", sizeof("collected")-1, (long)status.collected);
 	add_assoc_long_ex(return_value, "threshold", sizeof("threshold")-1, (long)status.threshold);
+	add_assoc_long_ex(return_value, "buf_size", sizeof("buf_size")-1, (long)status.buf_size);
```

Comment:
Could we name this field as `buffer_size`? When we renamed hundreds of parameters in PHP 8.0, we decided not to use abbreviations, unless they are either much shorter than the original name and/or trivial (e.g. `num`, `param`).

Reply:
Done.